client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/index.tsx

```diff
@@ -87,14 +92,15 @@ const designSetup: Step = function DesignSetup( { navigation, flow } ) {
 	const { data: themeDesigns = [] } = useDesignsBySite( site );
 
 	const staticDesigns = themeDesigns;
-	const generatedDesigns = useGeneratedDesigns();
+	const shuffledStaticDesigns = useMemo(
+		() => shuffle( staticDesigns.filter( ( design ) => ! isBlankCanvasDesign( design ) ) ),
+		[ staticDesigns, isForceStaticDesigns ]
```

Comment:
Why do we need to recompute the shuffled designs when `isForceStaticDesigns`?

Reply:
Good catch, forgot to remove it 😓